packages/common/src/dotlottie-player.ts

```diff
@@ -64,12 +64,21 @@ export interface DotLottieElement extends Element {
   __lottie?: AnimationItem | null;
 }
 
-export type RendererSettings = SVGRendererConfig & CanvasRendererConfig & HTMLRendererConfig;
-export type DotLottieConfig<T extends RendererType> = Omit<AnimationConfig<T>, 'container'> & {
-  mode?: PlayMode;
-  testId?: string | undefined;
+export const EXTRA_OPTIONS = {
+  count: 1,
+  direction: 1 as 1 | -1,
```

Comment:
i wonder why do we need this type assertion ? can't we set a type for the `EXTRA_OPTIONS`